weave-js/src/common/components/WBMenu.tsx

```diff
@@ -106,130 +100,69 @@ export const WBMenu = React.forwardRef<HTMLDivElement, WBMenuProps>(
       fontSize,
       lineHeight,
       highlightFirst,
-      highlighted: highlightedProp,
-      onChangeHighlighted,
       onSelect,
       onEsc,
       dataTest,
     },
     ref
   ) => {
-    const getDefaultHoverIndex = React.useCallback(() => {
-      let found = -1;
-      if (highlightFirst && options.length > 0) {
-        found = options.findIndex(opt => !opt.disabled);
-      } else {
-        found = options.findIndex(opt => selected === opt.value);
-      }
+    const [defaultHighlighted, defaultHighlightedIndex] = useMemo(() => {
+      if (!highlightFirst) return [undefined, undefined];
+      const foundIndex = options.findIndex(el => !el.disabled);
```

Comment:
before we also looked for the `selected === opt.value`, why don't we have to do that now?

Reply:
Good catch -- updating